web/src/DevServerWrapper.jsx

```diff
@@ -0,0 +1,112 @@
+/*
+ * Copyright (c) [2023] SUSE LLC
+ *
+ * All Rights Reserved.
+ *
+ * This program is free software; you can redistribute it and/or modify it
+ * under the terms of version 2 of the GNU General Public License as published
+ * by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
+ * more details.
+ *
+ * You should have received a copy of the GNU General Public License along
+ * with this program; if not, contact SUSE LLC.
+ *
+ * To contact SUSE LLC about this file by physical or electronic mail, you may
+ * find current contact information at www.suse.com.
+ */
+
+import React, { useEffect, useRef, useState } from "react";
+import { Button, Title, Text, EmptyState, EmptyStateIcon, EmptyStateBody } from "@patternfly/react-core";
+import { Center, Icon, Loading } from "~/components/layout";
+
+// path to any internal Cockpit component to force displaying the login dialog
+const loginPath = "/cockpit/@localhost/system/terminal.html";
```

Comment:
It looks like the path for the terminal component. What am I missing?

Reply:
Yes, it is the path to the terminal component. But if you are not logged in you will see the login screen instead. We could use path to any Cockpit component here, I just selected the terminal as we already use it so it should work.

Reply:
Ah, sorry. I understand it now. I think I didn't read the word "any" in the comment before. Please, excuse me.